src/AppInstallerCLICore/Commands/CompleteCommand.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+#include "Command.h"
+
+namespace AppInstaller::CLI
+{
+    // Command to enable tab completion scenarios, including allowing them to
+    // be context sensitive in their data output.
+    struct CompleteCommand final : public Command
+    {
+        CompleteCommand(std::string_view parent) : Command("complete", parent, Visibility::Hidden) {}

```

Comment:
>CompleteCommand(std::string_view parent) : Command("complete", parent, Visibility::Hidden) {} [](start = 8, length = 93)

So we are not going to release it as an experimental feature?

Reply:
No, since you have to opt in by setting up your shell to call into this, and it is otherwise out of the way, it isn't really necessary.

---
In reply to: [463771454](https://github.com/Microsoft/pkg/pull/521#discussion_r463771454) [](ancestors = 463771454)